plugin/src/main/java/com/h3xstream/findsecbugs/scala/XssMvcApiDetector.java

```diff
@@ -47,8 +47,8 @@ protected int getPriorityFromTaintFrame(TaintFrame fact, int offset)
             // Get the value of the content-type parameter
             Taint parameterTaint = fact.getStackValue(0);
 
-            if ( parameterTaint != null
-                    && (!parameterTaint.isSafe() || parameterTaint.getConstantValue().toLowerCase().equals(VULNERABLE_CONTENT_TYPE)) ) {
+            if ( !parameterTaint.isSafe()
```

Comment:
of simply : 
`if(!parameterTaint.isSafe() || VULNERABLE_CONTENT_TYPE.equalsIgnoreCase(parameterTaint.getConstantValue())) {`